lib/configuration/configuration_parser.js

```diff
@@ -0,0 +1,50 @@
+const Logger = require('../logger');
+const fsPromises = require('fs/promises');
+const clientConfiguration = require('./client_configuration');
+
+exports.parse = async function (configFilePath) {
+  const configFile = await tryToFind(configFilePath);
+  return configFile == null ? null : tryToParse(configFile);
+};
+
+function tryToFind (filePath) {
+  if (!filePath) {
+    return Promise.resolve(null);
+  }
+  return fsPromises.readFile(filePath, { encoding: 'utf8' })
+    .catch(err => {
+      const errorMessage = 'Finding easy logging configuration failed';
```

Comment:
it's easy loging configuratio or just client configuration?

Reply:
hard to say. I changed to "client configuration".